src/content/quickstart/kind/_index.md

```diff
@@ -58,12 +58,26 @@ You now have a Submariner environment that you can experiment with.
 
```

Comment:
This section is basically a duplicate of _verify_with_discovery.md_ - we should just include it as we do in other files.

Reply:
The kubeconfig paths used here is different since it is a kind deployment and hence we have did not use the include.